openlibrary/components/MergeUI/MergeTable.vue

```diff
@@ -113,6 +113,26 @@ export default {
             return records;
         },
 
+        async enhancedRecords(){
```

Comment:
To avoid someone in the future accidentally saving!

```suggestion
        /** The records with extra helpful metadata attached for display. Should NOT be save */
        async enhancedRecords(){
```